tests/integration/helpers.bash

```diff
@@ -14,7 +14,15 @@ BUSYBOX_IMAGE="$BATS_TMPDIR/busybox.tar"
 BUSYBOX_BUNDLE="$BATS_TMPDIR/busyboxtest"
 
 # hello-world in tar format
-HELLO_IMAGE="$TESTDATA/hello-world.tar"
+case $(dpkg --print-architecture) in
```

Comment:
If we use dpkg, non debian systems won't be able to run local integration tests, are there any other standard way to detect the osarch?

Reply:
`$(go env GOARCH)` seems like it would be a much better idea.

Reply:
OK. I will update it.